### DISCUSSION TITLE
2021.02.25 목 회의록

### DISCUSSION THREAD

**Discussion by hakju** [hakju/ios-cloud-notes#26] [General]
1. TIL 꼭! 쓰기 (CoreData, Respoder 등)
2. selfcam,,,,,,,,,
3. issues 전부 처리하기
4. 오늘 수업 예습하기 (iOS file system)
5. Initialization에 대해서 공부하기

**Comment by soleJin** [comment]
+ issues 처리
+ 수업 예습
+ selfcam?
